src/configuration-schema.js

```diff
@@ -243,6 +243,75 @@ export const configurationSchema = {
       },
     ],
   },
+  quizValidationCallback: {
```

Comment:
I thought about it, but I didn't like it. Yes, it looks clean if you think about the usage first, but there are several downsides imo. Currently, all configuration is done with the configuration attribute. Having this one thing configured by javascript is not consistent. This will require extra handling in documentation as well as in asserting that the user really gives us a function. Also, we can not set it to "required" anymore, which means that we always have to check whether it was set or not.

I agree that having a global function is ugly, and referencing it via name is even more ugly, but I think it's the better approach. But, if you'd still rather have it the other way, I'll do it.

Reply:
Yeah, you're right. This isn't the best solution either. So I'm fine with providing the callback as a global function the way you implemented it. 

But then I would suggest to nest quiz-related options in the configuration rather than having them both on root level. Besides, the callback should not be required as the quiz is an optional feature.

Reply:
check